packages/contracts-bedrock/deploy-config/devnetL1.json

```diff
@@ -41,10 +41,11 @@
   "governanceTokenOwner": "0xa0Ee7A142d267C1f36714E4a8F75612F20a79720",
   "eip1559Denominator": 50,
   "eip1559Elasticity": 6,
-  "l1GenesisBlockTimestamp": "0x64c811bf",
+  "l1GenesisBlockTimestamp": "0x64fb78c6",
   "l2GenesisRegolithTimeOffset": "0x0",
   "faultGameAbsolutePrestate": "0x41c7ae758795765c6664a5d39bf63841c71ff191e9189522bad8ebff5d4eca98",
   "faultGameMaxDepth": 30,
   "faultGameMaxDuration": 1200,
-  "systemConfigStartBlock": 0
-}
+  "systemConfigStartBlock": 0,
+  "permit2Address": "0x000000000022D473030F116dDEE9F6B43aC78BA3"
```

Comment:
instead of adding to the deploy-config, should we just inherit from the [constant](https://github.com/Uniswap/permit2/blob/79658f604fa017635177802a9e833b9065a13053/src/libraries/Permit2Lib.sol#L27) defined in Uniswap's code? since we assume Permit2 will always be deployed to the same addres

Reply:
updated after adding permit2 to devnet at deterministic address